webpack.config.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const { bundler } = require( '@ckeditor/ckeditor5-dev-utils' );
+const UglifyJsWebpackPlugin = require( 'uglifyjs-webpack-plugin' );
+
+module.exports = {
+	context: __dirname,
+
+	devtool: 'source-map',
+	performance: { hints: false },
+	externals: { react: 'React' },
```

Comment:
We should pass an object here to satisfy all environments: https://webpack.js.org/configuration/externals/#externals

Reply:
The build lands in the `dist` directory, which you will publish on the NPM. And then you can add it using e.g. `<script>` tag or load it via `import CKEditor from '@ckeditor/ckeditor5-react';`. So only we build this package, but it can be used in different ways.

Reply:
Sorry, I don't understand. 

There are two ways to use it and both ways are tested: (1) in `tests/ckeditor-classiceditor.jsx` we check the build and (2) the sample checks whether an editor works.